packages/noya-state/src/reducers/themeReducer.ts

```diff
@@ -361,28 +375,52 @@ export function themeReducer(
         });
       });
     }
-    case 'setThemeStyleName':
-    case 'setTextStyleName':
-    case 'setSwatchName': {
+    case 'setThemeStyleName': {
       const [, id, name] = action;
 
       const ids = typeof id === 'string' ? [id] : id;
 
       return produce(state, (draft) => {
-        const array =
-          action[0] === 'setSwatchName'
-            ? draft.sketch.document.sharedSwatches?.objects ?? []
-            : action[0] === 'setTextStyleName'
-            ? draft.sketch.document.layerTextStyles?.objects ?? []
-            : draft.sketch.document.layerStyles?.objects ?? [];
+        setComponentName(
+          ids,
+          name,
+          draft.sketch.document.layerTextStyles?.objects ?? [],
```

Comment:
Should this one be `.layerStyles` instead of `.layerTextStyles`?

Reply:
good catch! I must have missed it